demo/dynamic_image.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+    Copyright 2019 WerWolv
+    Copyright 2019 p-sam
+    Copyright 2020-2021 natinusala
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#include "dynamic_image.hpp"
+
+DynamicImage::DynamicImage()
```

Comment:
I would not call it "dynamic image", I don't find it very explicit

How about something like "RandomNoiseImage"? Or RandomRGBAImage?